src/style.css

```diff
@@ -0,0 +1,79 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-image: url(./images/star.svg);
+  background-position: center center;
+  background-repeat: no-repeat;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  margin-left: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  margin-left: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  margin-left: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
```

Comment:
There is no need to duplicate all the properties. You should change only `background-image`.
All other properties remain the same